electron/src/preload/preload.ts

```diff
@@ -0,0 +1,16 @@
+// See the Electron documentation for details on how to use preload scripts:
+// https://www.electronjs.org/docs/latest/tutorial/process-model#preload-scripts
+import { contextBridge, ipcRenderer } from 'electron';
+
+contextBridge.exposeInMainWorld('electron', {
+  enabled: true,
+  getPreloadPath: () => ipcRenderer.sendSync('get-preload-path'),
```

Comment:
Are we actually using this anywhere?

Reply:
Was for testing the ipc system - removed.